core-services/secret-mirroring/_mapping.yaml

```diff
@@ -566,3 +566,9 @@ secrets:
   to:
     namespace: coreos
     name: coreos-registry-registry.ci.openshift.org
+- from:
+    namespace: hypershift-clusterbot
+    name: admin-credentials
```

Comment:
This doesn't exist.
/hold

Reply:
Something I don't understand about the secret mirroring — the namespace itself is defined in this PR (`core-services/secrets/hypershift-clusterbot/hypershift-clusterbot.yaml`). What is the order of operations to make this happen? And after the namespace is created (by doing what...?) does that secret have to exist before I commit the mapping? If so, what happens if the secret is ever deleted? etc.

Reply:
Good point. Yeah, chicken egg problem. The whole mechanism was never explicitly designed but organically grew over time. We are working on a replacment but its not done yet.
/hold cancel
/lgtm
/approve